lib/publiccloud/provider.pm

```diff
@@ -376,7 +376,7 @@ sub terraform_apply {
     $cmd .= "-out myplan";
     record_info('TFM cmd', $cmd);
 
-    assert_script_run($cmd, $terraform_timeout);
+    script_retry($cmd, timeout => $terraform_timeout, delay => 3, retry => 6);
```

Comment:
Perhaps we should consider running a `terraform destroy` between retries to ensure that there are no orphaned resources.

Reply:
What about this?
```perl
script_retry($cmd.' || (terraform destroy; false)', timeout => $terraform_timeout, delay => 3, retry => 6);
```

Reply:
I can see, that we are only retrying the `terraform plan`, so I suppose the `destroy` is not needed here?